### PR TITLE
web server for nn with external redis

### PR DESCRIPTION
**Pull request by YaphetKG**
NN deployment that assumes external redis server. 
This is for the changes to NN in redis cluster support.

### CODE REVIEW

File: helm/node-normalization-web-server/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        paths:
+          - path: "/1.1(/|$)(.*)"
+            backend:
+              serviceName: {{ include "node-normalization-webserver.fullname" . }}-web-service-trapi-1-1
+              servicePort: {{ .Values.web.port }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "node-normalization-webserver.fullname" . }}-web-service-trapi-1-1-2
```
Comment (PhillipsOwen) [inline helm/node-normalization-web-server/templates/ingress.yaml:25]: are we still using that -trapi-1-1 on everything? i am trying to make the app version, image version, and repo tag all be the same across all the projects i touch now.

**Review by PhillipsOwen — COMMENTED** [review]
are we still using that -trapi-1-1 on various tags? i am trying to make the app version, image version, and repo tag all be the same across all the projects i touch now.

**Comment by YaphetKG** [comment]
> 
> 
> are we still using that -trapi-1-1 on various tags? i am trying to make the app version, image version, and repo tag all be the same across all the projects i touch now.

I think we can correct the refs of trapi-1-1 from service names, deployment spec etc... As for the images will be using the Semver tags as you have suggested over our slack conversation

**Comment by YaphetKG** [comment]
@PhillipsOwen i've removed all the trapi 1-1 refs from the service and deployment spec. the only place 1.1 is present atm is on the ingress path and the root path of the one of the deployments.

**Review by PhillipsOwen — APPROVED** [review]
looks good!

**Pull request merged**